lib/menu.rb

```diff
@@ -0,0 +1,37 @@
+# This class holds all of a resturants dish options
+class Menu
+
+  def initialize
+    @list = []          # holds dishes
+    @loaded_dish = nil  # needed so that find dish returns a dish
+  end
+
+  def add_dish(dish)
+    @list << dish # adds dish to the list array
+  end
+
+  def list_dishes
```

Comment:
Not sure if it's worth having a private method when the public method only calls the private method. In a case like this I would expect the list_dishes method to include the code for listing the dishes. I'd use a private method for something that I didn't want the user to be able to do, needs to be called by the code in a public method.